scalafmt-tests/shared/src/test/scala/org/scalafmt/cli/CliTest.scala

```diff
@@ -295,8 +298,14 @@ trait CliTestBehavior {
           "--config-str",
           s"""{version="$version",style=IntelliJ}""",
         )
-        intercept[IOException] {
-          Cli.exceptionThrowingMainWithOptions(args, baseCliOptions)
+        if(PlatformCompat.isNativeOnWindows()) {
```

Comment:
formatting issue (needs a space). you should run `./scalafmt` in your local checkout directory before amending the commit.